### PR TITLE
 [docs] translate mysql cdc doc to Chinese.#1391

### PR DESCRIPTION
**Pull request by rookiegao**
This PR Main solutions #1391

### CODE REVIEW

**Review by leonardBang — APPROVED** [review]
Thanks @rookiegao for the contribution, LGTM. I also help improve a little.

File: docs/content/connectors/mysql-cdc.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ In order to setup the MySQL CDC connector, the following table provides dependen
 
 Download [flink-sql-connector-mysql-cdc-2.3-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mysql-cdc/2.3-SNAPSHOT/flink-sql-connector-mysql-cdc-2.3-SNAPSHOT.jar) and put it under `<FLINK_HOME>/lib/`.
 
-**Note:** flink-sql-connector-mysql-cdc-XXX-SNAPSHOT version is the code corresponding to the development branch. Users need to download the source code and compile the corresponding jar. Users should use the released version, such as [flink-sql-connector-mysql-cdc-XXX.jar](https://mvnrepository.com/artifact/com.ververica/flink-connector-mysql-cdc), the released version will be available in the Maven central warehouse. 
+**Note:** flink-sql-connector-mysql-cdc-XXX-SNAPSHOT version is the code corresponding to the development branch. Users need to download the source code and compile the corresponding jar. Users should use the released version, such as [flink-sql-connector-mysql-cdc-2.2.1.jar](https://mvnrepository.com/artifact/com.ververica/flink-connector-mysql-cdc), the released version will be available in the Maven central warehouse.
```
Comment (rookiegao) [inline docs/content/connectors/mysql-cdc.md:34]: https://mvnrepository.com/artifact/com.ververica/flink-connector-mysql-cdc   ->
https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-mysql-cdc

Other connector documents also need to be modified accordingly

File: docs/content/connectors/mysql-cdc(ZH).md [hunk 2]
```diff
@@ -0,0 +1,882 @@
...
+       MULTILINESTRING<br>
+       MULTIPOLYGON<br>
+       GEOMETRYCOLLECTION<br>
+      </td>
+      <td>
+        STRING
+      </td>
+      <td>
+      MySQL 中的空间数据类型将转换为具有固定 Json 格式的字符串。
+      请参考 MySQL<a href="# 空间数据类型映射">MySQL 空间数据类型映射</a> 章节了解更多详细信息。
```
Comment (rookiegao) [inline docs/content/connectors/mysql-cdc(ZH).md:810]: repeated 'mysql'

**Comment by rookiegao** [comment]
@leonardBang I have put forward some suggestions for improving the current document. If you agree, I will make corresponding changes at night, because I don't have a suitable environment to deal with it during the day

**Comment by leonardBang** [comment]
> @leonardBang I have put forward some suggestions for improving the current document. If you agree, I will make corresponding changes at night, because I don't have a suitable environment to deal with it during the day

I've addressed the your suggestions.

File: docs/content/connectors/oracle-cdc.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ In order to setup the Oracle CDC connector, the following table provides depende
 
 Download [flink-sql-connector-oracle-cdc-2.3-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-oracle-cdc/2.3-SNAPSHOT/flink-sql-connector-oracle-cdc-2.3-SNAPSHOT.jar) and put it under `<FLINK_HOME>/lib/`.
 
-**Note:** flink-sql-connector-oracle-cdc-XXX-SNAPSHOT version is the code corresponding to the development branch. Users need to download the source code and compile the corresponding jar. Users should use the released version, such as [flink-sql-connector-oracle-cdc-XXX.jar](https://mvnrepository.com/artifact/com.ververica/flink-connector-oracle-cdc), the released version will be available in the Maven central warehouse.
+**Note:** flink-sql-connector-oracle-cdc-XXX-SNAPSHOT version is the code corresponding to the development branch. Users need to download the source code and compile the corresponding jar. Users should use the released version, such as [flink-sql-connector-oracle-cdc-2.2.1.jar](https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-oceanbase-cdc), the released version will be available in the Maven central warehouse.
```
Comment (rookiegao) [inline docs/content/connectors/oracle-cdc.md:27]:
```suggestion
**Note:** flink-sql-connector-oracle-cdc-XXX-SNAPSHOT version is the code corresponding to the development branch. Users need to download the source code and compile the corresponding jar. Users should use the released version, such as [flink-sql-connector-oracle-cdc-2.2.1.jar](https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-oracle-cdc), the released version will be available in the Maven central warehouse.
```

**Comment by rookiegao** [comment]
The exciting work is finished. Thank you for your guidance @leonardBang

**Comment by leonardBang** [comment]
> The exciting work is finished. Thank you for your guidance @leonardBang

Thanks @rookiegao for the great work, this is the first Chinese connector document, I'm pretty sure your contribution will help Chinese speaking user a lot.

**Comment by rookiegao** [comment]
> @rookiegao Would you also like to backport the PR to `release-2.2` branch? as we recommended user to look up the latest release version instead of the master version.

of course

**Pull request closed without merging**